static/app/views/insights/sessions/charts/errorFreeSessionsChart.tsx

```diff
@@ -11,7 +11,8 @@ export default function ErrorFreeSessionsChart() {
 
   return (
     <InsightsLineChartWidget
-      title={t('Error Free Session Rate')}
+      title={t('Error Free Sessions')}
+      description={t('Percent of healthy sessions out of total sessions.')}
```

Comment:
The percent of sessions that finished without a single error occurring, see, see [Session status](https://docs.sentry.io/product/releases/health/#session-status).